pallets/proxy/src/lib.rs

```diff
@@ -98,7 +110,7 @@ pub mod pallet {
         // create an encoded version of the call
         let mut encoded_call = Vec::<u8>::new();
         // first byte is the pallet id on the destination chain
-        encoded_call.push(T::AcurastPalletId::get());
```

Comment:
Why did this change?

Reply:
We should use `T::AcurastPalletId::get()` preferably.

Reply:
We are still using `T::AcurastPalletId::get()` and `T::AcurastMarketplacePalletId::get()` to choose where to send the call, but now we pass that over a newly introduced parameter `palletId` to this very function

Reply:
In my opinion we should have a single getter that either gets `pallet_acurast` or `pallet_acurast_merketplace` id's, not both simultaneously.

I am fine ignoring this improvement on this MR, but we should look at it once the MR gets merged.

Things that can be done:

1. Having 2 proxy implementations, one for the standalone and other for the merketplace.
2. Add a feature to disable the advertisement logic.